### PR TITLE
Issue#186 ensure user is logged out from opensrp

### LINKED ISSUE #186: Ensure superset user is logged out after logging out
When you click the log out button, we need to ensure that we log out of superset by calling the `deAuthZ` method in SupersetConnector

### PR DESCRIPTION
**Pull request by FrankApiyo**
Fixes #186

### CODE REVIEW

File: src/components/page/Header/index.tsx [hunk 1]
```diff
@@ -255,6 +260,17 @@ export class HeaderComponent extends React.Component<HeaderProps, State> {
     );
   }
 
+  private handleLogout() {
+    const logoutURL: string = 'https://reveal-stage.smartregister.org/opensrp/logout.do';
```
Comment (p-netm) [inline src/components/page/Header/index.tsx]: lets put this in the env file,

Comment (FrankApiyo) [reply src/components/page/Header/index.tsx]: we will indeed.

**Comment by FrankApiyo** [comment]
> Am iffy about the sideeffects this imposes on the ui, will have to wait and see what @cKellyDesign thinks of it.

I think we could make the time taken before the window we open in closed automatically longer... then it'll be less weird.

File: src/components/page/Header/index.tsx (hunk 1 above)
Comment (cKellyDesign) [reply src/components/page/Header/index.tsx]: > lets put this in the env file

agreed, let's pipe in in from `.env` through `env.ts` and import it here, and let's add it to `.env.sample` too

**Comment by FrankApiyo** [comment]
now that code is tested :ok_hand:

**Pull request merged**